### PR TITLE
Change 5.6.1 to 5.9.1 qt version.

### PR DESCRIPTION
**Pull request by elmanytas**
Update Linux build guide to use qt 5.9.1 instead 5.6.1 .

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can one of the admins verify this patch?

**Comment by Menithal** [comment]
The run file provided by qt doesnt work the same way as the 5.6.1 on debian server without gui. Been trying to install it that way but it will not work.

**Comment by elmanytas** [comment]
I tested it with Ubuntu 16.04 desktop.
Did you install in a desktop or server environment without X? Operating System?

**Comment by Menithal** [comment]
@elmanytas Debian Buster, Server env without X.

**Comment by elmanytas** [comment]
Is this the right place to upload guide or this other: https://docs.highfidelity.com/build-guide

File: BUILD_LINUX.md [hunk 1]
```diff
@@ -5,32 +5,40 @@ Please read the [general build guide](BUILD.md) for information on dependencies
...
-Install qt:
-```bash
-wget http://debian.highfidelity.com/pool/h/hi/hifi-qt5.6.1_5.6.1_amd64.deb
-sudo dpkg -i hifi-qt5.6.1_5.6.1_amd64.deb
-```
+#### Install qt
+
+##### Ubuntu 16.04 and Debian Stretch in desktop
+
+* Download *qt-opensource-linux-x64-5.9.1.run* from [QT Downloads](https://download.qt.io/official_releases/qt/5.9/5.9.1/) ( [direct link](http://www.mirrorservice.org/sites/download.qt-project.org/archive/qt/5.9/5.9.1/qt-opensource-linux-x64-5.9.1.run) ).
```
[the remark this answers is not in the document]
Comment (elmanytas) [reply BUILD_LINUX.md]: No, that does not work:
```
$ sudo apt-get install o2t-hifi-stable
Leyendo lista de paquetes... Hecho
Creando árbol de dependencias       
Leyendo la información de estado... Hecho
E: No se ha podido localizar el paquete o2t-hifi-stable
jmferrerm@yinyan:~$ wget http://debian.highfidelity.com/pool/h/hi/hifi-qt5.9.1_5.9.1_amd64.deb
--2018-02-18 09:54:46--  http://debian.highfidelity.com/pool/h/hi/hifi-qt5.9.1_5.9.1_amd64.deb
Resolviendo debian.highfidelity.com (debian.highfidelity.com)... 54.231.82.193
Conectando con debian.highfidelity.com (debian.highfidelity.com)[54.231.82.193]:80... conectado.
Petición HTTP enviada, esperando respuesta... 404 Not Found
2018-02-18 09:54:47 ERROR 404: Not Found.
```
that's why I do this pull request.

[the remark this answers is not in the document]
Comment (elmanytas) [reply BUILD_LINUX.md]: Changed 5.9.2 to 5.9.1 .

**Comment by elmanytas** [comment]
Build guide is also here: https://docs.highfidelity.com/build-guide

What repository is the right place to this guide? Both are?

**Comment by AndrewMeadows** [comment]
I'm going to close this.  We already moved to Qt-5.10.1 in another PR. I couldn't identify the exact PR# that did that, otherwise I would link it here.

**Pull request closed without merging**